mk-files/release.mk

```diff
@@ -148,3 +150,32 @@ download-licenses:
 publish-installer:
 	$(aws-authenticate) && \
 	$(call dry-run,aws s3 cp install.sh $(S3_BUCKET_PATH)/confluent-cli/install.sh --acl public-read)
+
+.PHONY: update-muckrake
+update-muckrake:
+	$(eval DIR=$(shell mktemp -d))
+	$(eval CLI_RELEASE=$(DIR)/cli-release)
+	$(eval MUCKRAKE=$(DIR)/muckrake)
+
+	git clone git@github.com:confluentinc/cli-release.git $(CLI_RELEASE) && \
+	cd $(CLI_RELEASE) && \
+	version=$$(ls release-notes | $(SED) "s/.json$$//" | sort --version-sort | tail -1) && \
+	git clone git@github.com:confluentinc/muckrake.git $(MUCKRAKE) && \
```

Comment:
muckrake is being cloned under cli-release?

Reply:
Yup, we need to clone it before we can open a PR.

Reply:
i'm confused it looks like muckrake is being clones inside of cli-release, but from `$(eval CLI_RELEASE=$(DIR)/cli-release) $(eval MUCKRAKE=$(DIR)/muckrake)` they're at the same level of dir

Reply:
The second argument to `git clone` is the directory that we're cloning into!

Reply:
oh haha